cilium/cmd/bpf_auth_list.go

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/maps/auth"
+	"github.com/cilium/cilium/pkg/policy"
+)
+
+type authEntry struct {
+	LocalIdentity  uint32
+	RemoteIdentity uint32
+	RemoteNodeID   uint16
+	AuthType       uint8
+	Expiration     int64
```

Comment:
We'll want to format this as human readable time at some point, but this is fine for now.

Reply:
agree. for the time being it's really more to inspect the content of the map.